.github/release-drafter.yml

```
@@ -0,0 +1,49 @@
+name-template: "MQTProblemSolver $RESOLVED_VERSION Release"
```

```suggestion
name-template: "MQT ProblemSolver $RESOLVED_VERSION Release"
```
Just a minor nit: Wouldn't you want to include the space here for a better look?